app/views/mvc/questions.html.erb

```diff
@@ -1,10 +1,25 @@
 <h1>What part of the MVC framework would deal with implementing the requested functionality?</h1>
-<h3>1)If shipping address is Colorado, apply 6% sales tax to all orders</h3>
-<p>Answer</p>
-<h3>2)Do not allow shipments to Florida</h3>
-<h3>3)Require user to login before checking out</h3>
-<h3>4)The background color of the webpage should be light green</h3>
-<h3>5)If the user tries to add an out of stock item to their cart, they should be redirected and see an error (hint: more than one; explain thinking)</h3>
-<h3>6)The user should have a drop down to allow them to add a quantity of an item to the cart</h3>
-<h3>7)If the user's shopping cart is empty, redirect them to the main page</h3>
-<h3>8)If the user requests quantity 5, but only three are available, that row in the shopping cart should be highlighted in red</h3>
\ No newline at end of file
+
+<h3>1) If shipping address is Colorado, apply 6% sales tax to all orders</h3>
+<p>Model</p>
+
+<h3>2) Do not allow shipments to Florida</h3>
+<p>Model</p>
+
+<h3>3) Require user to login before checking out</h3>
+<p>Controller</p>
+
+<h3>4) The background color of the webpage should be light green</h3>
+<p>View</p>
+
+<h3>5) If the user tries to add an out of stock item to their cart, they should be redirected and see an error</h3>
+<p>The Model will tell the Controller that the item is out of stock, the Controller will set an error message, and the View shows the error message.</p>
+
+<h3>6) The user should have a drop down to allow them to add a quantity of an item to the cart</h3>
+<p>The View shows the drop down, and the Controller is what the user interacts with to add the quantity.</p>
+
+<h3>7) If the user's shopping cart is empty, redirect them to the main page</h3>
+<p>The Controller sees that the shopping cart is empty and tells the View to show the main page.</p>
```

Comment:
The shopping cart is a model, so it would know if it's empty or not.  Redirecting is a controller thing.